dapps/marketplace/src/utils/store.js

```diff
@@ -4,15 +4,15 @@ import unset from 'lodash/unset'
 
 const memStore = {}
 
-export default function store(type = 'localStorage') {
+export default function store(type = 'localStorage', rootKey = 'uiState') {
```

Comment:
I wanted to store all the settings in one bunch, but maybe `uiState` doesn't reflect the intention for all sorts of settings. So here I tried to extend this interface, but also planning to dive deeper in Apollo framework, to see how it can handle remote and local data uniform way.